### ISSUE TITLE
An option to enable / disable the log on the front page 

### ISSUE THREAD

**Issue by henrikkoch** [Part-DB/Part-DB-server#470]
Just for the view - I think the front page "disturbs" me as a lot of it is a list of all the changes that has been done. 

Could it be a feature that you can enable/disable - or just have another page that is showing the log

And thanks for a great tool!!

By the way, I'm using 1.10.1

**Comment by jbtronics** [comment]
Im currently working on a system, where you can configure (most) of Part-DB settings using the web frontend.

If that is existing, then settings like this can be added.

**Comment by mkne** [comment]
Issue seems to be still open with the new settings system in version 2.0.2?

**Comment by jbtronics** [comment]
Yes the 2.0 versions just implemented existing options so far. But now it should just be a few lines of code to implement this.

**Comment by jbtronics** [comment]
This is possible with the latest commits.
